File: src/clr2jvm/compat-lib/Java/System/Windows/Forms/MessageBoxIcon.java
/* Copyright (c) 2002-2011 by XMLVM.org
 *
 * Project Info:  http://www.xmlvm.org
 *
 * This program is free software; you can redistribute it and/or modify it
 * under the terms of the GNU Lesser General Public License as published by
 * the Free Software Foundation; either version 2.1 of the License, or
 * (at your option) any later version.
 *
 * This library is distributed in the hope that it will be useful, but
 * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
 * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public
 * License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public
 * License along with this library; if not, write to the Free Software
 * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301,
 * USA.
 */

package System.Windows.Forms;
import System.Enum;
public class MessageBoxIcon extends Enum
{
	final public static int None = 0;
	final public static int Hand = 16;
	final public static int Error = 16;
	final public static int Stop = 16;
	final public static int Question = 32;
	final public static int Exclamation = 48;
	final public static int Warning = 48;
	final public static int Asterisk = 64;
	final public static int Information = 64;
	public int value__;
	public MessageBoxIcon()
	{
		this.value__ = 0;
	}
	public MessageBoxIcon(int toSet)
	{
		this.value__ = toSet;
	}
	public static MessageBoxIcon __BOX(int toBox)
	{
		return new MessageBoxIcon(toBox);
	}
}

File: src/xmlvm2js/System/Windows/Forms/MessageBoxIcon.js
/* Copyright (c) 2002-2011 by XMLVM.org
 *
 * Project Info:  http://www.xmlvm.org
 *
 * This program is free software; you can redistribute it and/or modify it
 * under the terms of the GNU Lesser General Public License as published by
 * the Free Software Foundation; either version 2.1 of the License, or
 * (at your option) any later version.
 *
 * This library is distributed in the hope that it will be useful, but
 * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
 * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public
 * License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public
 * License along with this library; if not, write to the Free Software
 * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301,
 * USA.
 */

qx.Class.define("System_Windows_Forms_MessageBoxIcon", {
	extend :System_Enum,
	construct : function(in_val) {
		this._value__ = in_val;
	},
	statics : {
		None :0,
		Hand :16,
		Error :16,
		Stop :16,
		Question :32,
		Exclamation :48,
		Warning :48,
		Asterisk :64,
		Information :64,
		___BOX___int : function(in_val) {
			return new System_Windows_Forms_MessageBoxIcon(in_val);
		}
	},
	members : {
		_value__ :0
	}
});
